src/App/Pages/Vault/AddEditPageViewModel.cs

```diff
@@ -297,7 +297,7 @@ public async Task<bool> LoadAsync(AppOptions appOptions = null)
                 if (org.Enabled && org.Status == OrganizationUserStatusType.Confirmed)
                 {
                     OwnershipOptions.Add(new KeyValuePair<string, string>(org.Name, org.Id));
-                    if (policies != null && org.UsePolicies && !org.IsAdmin && AllowPersonal)
+                    if (policies != null && org.UsePolicies && !org.canManagePolicies && AllowPersonal)
```

Comment:
Assuming the idea here is that rather than `admin`, if the user has permissions to manage policies, they could just disable this policy anyway and do this (similar to an `admin` or `owner` before). Makes sense 👍🏻